File: local/HDR__/c_src/average.c
#include <stdio.h>
int main(void)
{
	int math, eng, kor;
	
	printf("math, eng, kor\n");
	scanf("%d %d %d", &math, &eng, &kor);
	
	int sum = math + eng + kor;
	double avg = (double)sum / 3.0;

	printf("avg : %.2f\t sum : %d\n", avg, sum);
	return 0;
}

File: local/HDR__/python/average.py
#report = input("국, 영, 수 :")
#list = report.split()
#sum = int(list[0]) + int(list[1]) + int(list[2])

# kor,eng,mat = list
# sum = int(kor) +int(eng) + int(mat)

list = list(map(int, input("국, 영, 수 :").split()))

print("총점 : ", sum(list), "평균 : ", sum(list)/len(list))
